Sources/Swift/Integrations/UserFeedback/Configuration/SentryUserFeedbackFormConfiguration.swift

```diff
@@ -41,16 +47,10 @@ public class SentryUserFeedbackFormConfiguration: NSObject {
     
     /**
      * The label shown next to an input field that is required.
-     * - note: Default: `"(required)"`
+     * - note: Default: `"(Required)"`
      */
     public var isRequiredLabel: String = "(Required)"
     
-    /**
-     * The message displayed after a successful feedback submission.
-     * - note: Default: `"Thank you for your report!"`
-     */
-    public var successMessageText: String = "Thank you for your report!"
```

Comment:
This was unused anywhere. the ux doesn't show any kind of success view by default, just dismisses the form. we can iterate later if we decide we want that. or maybe make it optional, and only display something if it's nonnil